src/main/java/christmas/exception/ErrorCode.java

```diff
@@ -0,0 +1,17 @@
+package christmas.exception;
+
+public enum ErrorCode {
+    PREFIX("[ERROR] "),
```

Comment:
에러 상황에 대한 메시지를 필드로 가지는 다른 상수와 달리 이 부분은 출력 형식에 관련된 `String`인데요.
이 부분은 `enum` 멤버가 아닌 상수로 존재하는 것이 조금 더 자연스럽다고 생각됩니다!

```suggestion
    private static final String PREFIX = "[ERROR] ";
```